components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java

```diff
@@ -506,7 +507,8 @@ public APIDefinitionValidationResponse validateAPIDefinition(String apiDefinitio
             OpenAPI openAPI = parseAttemptForV3.getOpenAPI();
             io.swagger.v3.oas.models.info.Info info = openAPI.getInfo();
             OASParserUtil.updateValidationResponseAsSuccess(validationResponse, apiDefinition, openAPI.getOpenapi(),
-                    info.getTitle(), info.getVersion(), null, info.getDescription());
+                    info.getTitle(), info.getVersion(), null, info.getDescription(),
+                    openAPI.getServers().get(0).getUrl());
```

Comment:
This may cause null pointer exception, ArrayIndexOutofBound exception if the servers are not defined. 

Need to check whether the openAPI.getServers() method for null.